packages/slate-lists/src/registry.ts

```diff
@@ -0,0 +1,27 @@
+import type { Editor } from 'slate';
+
+import type { ListsSchema } from './types';
+
+const EDITOR_LISTS_SCHEMA = new WeakMap<Editor, ListsSchema>();
```

Comment:
I've tried this as an experiment to achieve the Lists plugin functionality without mutating the `Editor` instance. This will become helpful when/if we refactor editor extensions to be linked dynamically (CARE-1802).